test/distributed/_pipeline/sync/test_pipe.py

```diff
@@ -525,46 +485,55 @@ def test_named_children():
         model.a
 
 
-def test_recommend_auto_balance():
-    with pytest.raises(ValueError, match="torch.distributed._pipeline.sync.balance"):
-        # balance is required
-        Pipe(nn.Sequential())
-
-    with pytest.raises(ValueError, match="torch.distributed._pipeline.sync.balance"):
-        # module and sum of balance have differen length (module: 0, sum of balance: 1)
-        Pipe(nn.Sequential(), [1])
-
-    with pytest.raises(ValueError, match="torch.distributed._pipeline.sync.balance"):
-        # module and sum of balance have different length (module: 2, sum of balance: 1)
-        Pipe(nn.Sequential(nn.Linear(1, 1), nn.Linear(1, 1)), [1])
-
-
 def test_verify_module_non_sequential():
     with pytest.raises(TypeError, match="module must be nn.Sequential to be partitioned"):
-        Pipe(nn.Module(), [1])
+        Pipe(nn.Module())
 
 
 def test_verify_module_duplicate_children():
     conv = nn.Conv2d(3, 3, 1)
     model = nn.Sequential(conv, conv)
 
     with pytest.raises(ValueError, match="module with duplicate children is not supported"):
-        Pipe(model, [1, 1])
+        Pipe(model)
 
 
 @skip_if_no_cuda
-def test_verify_module_duplicate_parameters_on_distinct_devices():
+def test_verify_module_params_on_same_device():
     class Surrogate(nn.Module):
-        def __init__(self, module):
+        def __init__(self, param1, param2):
             super().__init__()
-            self.module = module
-
-    conv = nn.Conv2d(3, 3, 1)
-    model = nn.Sequential(Surrogate(conv), Surrogate(conv))
-
-    with pytest.raises(ValueError, match="module with duplicate parameters on distinct devices is not supported"):
-        Pipe(model, [1, 1], devices=["cpu", "cuda"])
+            self.param1 = param1
+            self.param2 = param2
+
+    conv1 = nn.Conv2d(3, 3, 1)
+    conv2 = nn.Conv2d(3, 3, 1)
+    model = nn.Sequential(Surrogate(conv1, conv2.cuda()))
+
+    with pytest.raises(
+        ValueError,
+        match=r'should have all parameters on a single device, please use .to\(\)'
```

Comment:
This message matching has changed and was failing lint earlier to the escape sequence `\(`